library/classes/InsuranceNumbers.class.php

```diff
@@ -21,12 +21,13 @@ class InsuranceNumbers extends ORDataObject
                                                         "1D" => "Medicaid Provider Number","1G" => "Provider UPIN Number","1H" => "Champus Identification Number","1J" => "Facility ID Number",
```

Comment:
are the changes in this InsuranceNumbers.class.php script related to the phq9 form?

Reply:
this could be a new PR so prob best to remove this file

Reply:
hi @ruthkonyn just remove the changes here and commit and push

easy way to do this is just to cp the file that's in master branch here

Reply:
i'm unwilling to replace the InsuranceNumbers.class.php script with the master version in mark sarnov's fork, as i've not worked on this. I have asked mark to look at the file, but have had no answer as yet. 

 @msarnov are you able to help with this? 
my understanding is that to get the phq9 form accepted we need to revert to the master version of InsuranceNumbers.class.php and then do a PR - once that's done then the changes to the class file can be remade and a new PR done for the insurance numbers. thanks everyone.

Reply:
ok it's no biggie @ruthkonyn, we could add the change here and link an issue to it

here's a [PR tip](https://github.com/openemr/openemr/pull/5175#issuecomment-1258146500) for @msarnov

Reply:
removed the pesky whitespace there by editing it directly so the styling should now pass

Reply:
thanks @stephenwaite  (hmmm i tried that and couldn't see the white space, hey ho)  - do i need to do another pr?

Reply:
woo hoo - that worked - all tests passed - many thanks @stephenwaite

Reply:
Hi all. Sorry for the delay. 
The file InsuranceNumbers.class.php  is not related to the PHQ9 form in any way, shape, or form. It was a modification that I made to the file for my operational openEMR setup. The BCBS payor in my area (and I suspect others) require a taxonomy code for processing X12 claims and there was no place to put one in the native version of openEMR. I was not sure how others had accomplished the Taxonomy code addition, so I added a new entry in the array. 
If that is fouling up the acceptance of PHQ9, I can edit it out.

Reply:
no prob, we can include it, thanks @msarnov